Khan/wonder-blocks
# [fix:button,dropdown,icon-button] WB-478: Remove special highlight style on mobile

## Button, ActionMenu, IconButton
- Fixed: Removed blue highlight style shown when the user focuses/clicks a button on mobile devices.

### Test plan
1. Open the following links on a mobile device:
    - Button: https://deploy-preview-417--wonder-blocks.netlify.com/#!/Button/1
    - ActionMenu: https://deploy-preview-417--wonder-blocks.netlify.com/#!/ActionMenu/1
    - IconButton: https://deploy-preview-417--wonder-blocks.netlify.com/#!/IconButton/1
2. Click on any button on the examples.
3. Make sure the highlight don't appear anymore (this happens when the focus is activated for this clicked button).
